src/components/form-fields/text-field.tsx

```
@@ -0,0 +1,57 @@
+      <Input
+        type={type}
+        placeholder={placeholder}
+        disabled={disabled}
+        autoFocus={autoFocus}
+        name={name}
+        id={name}
+        defaultValue={defaultValue}
+        readOnly={readOnly}
+        aria-invalid={!!(error && error[name])}
+      />
+      {error && error[name] && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)